internal/npm_install/test/wksp/golden/jasmine/BUILD.bazel.golden

```diff
@@ -0,0 +1,10 @@
+
+package(default_visibility = ["//visibility:public"])
+alias(
+    name = "jasmine",
+    actual = "//node_modules/jasmine",
+)
+alias(
+    name = "jasmine__bin",
```

Comment:
what if jasmine had two `bin` entries?

Reply:
Would be ‘<other>__bin’ target in the jasmine package